hw/ip/lc_ctrl/data/lc_ctrl_state.hjson

```diff
@@ -7,9 +7,9 @@
 // $ ./util/design/gen-lc-state-enc.py
 
 {
-    // Seed to be used for generation of partition item default values.
+    // 256 bit seed to be used for generation of partition item default values.
     // Can be overridden on the command line with the --seed switch.
-    seed: 10167336684108184581
+    seed: 40182201019264397688411770949626922549663256047001778394918990008320537410392
```

Comment:
How are these seeds generated?

Reply:
I've used `secrets.py` module which uses the entropy source from the operating system.

Reply:
Maybe you can add a comment to this file saying how you've generated this (i.e. what Python commands did you use).

Reply:
Good idea. I've added comments to hjson files